docs/ops/activation/Mish.md

```diff
@@ -0,0 +1,48 @@
+## Mish <a name="Mish"></a>
+
+**Versioned name**: *Mish-4*
+
+**Category**: *Activation*
+
+**Short description**: Mish a Self Regularized Non-Monotonic Neural Activation Function.
+
+**Detailed description**: [Reference](https://www.tensorflow.org/addons/api_docs/python/tfa/activations/mish)
```

Comment:
Please, do not provide the reference as a description. Instead, provide the textual description.